MiniApp/Classes/core/RealMiniApp.swift

```diff
@@ -272,7 +273,8 @@ internal class RealMiniApp {
         }
         metaDataDownloader.getMiniAppMetaInfo(miniAppId: appId,
                                               miniAppVersion: version,
-                                              apiClient: self.miniAppClient) { (result) in
+                                              apiClient: self.miniAppClient,
+                                              languageCode: languageCode ?? "") { (result) in
```

Comment:
Shouldn't we use the default locale instead of `""` ?

```suggestion
                                              languageCode: languageCode ?? NSLocale.current.languageCode) { (result) in
```